ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -308,13 +308,15 @@ - (NSURL *)composeBrokerRequest:(ADAuthenticationError* __autoreleasing *)error
       @"username_type"  : _requestParams.identifier ? [_requestParams.identifier typeAsString] : @"",
       @"username"       : _requestParams.identifier.userId ? _requestParams.identifier.userId : @"",
       @"force"          : _promptBehavior == AD_FORCE_PROMPT ? @"YES" : @"NO",
+      @"skip_cache"     : _skipCache ? @"YES" : @"NO",
       @"correlation_id" : _requestParams.correlationId,
 #if TARGET_OS_IPHONE // Broker Message Encryption
       @"broker_key"     : base64UrlKey,
 #endif // TARGET_OS_IPHONE Broker Message Encryption
       @"client_version" : adalVersion,
       BROKER_MAX_PROTOCOL_VERSION : @"2",
       @"extra_qp"       : _queryParams ? _queryParams : @"",
+      @"claims"         : _claims ? _claims : @"",
```

Comment:
At some point we should do a pass to remove this sort of logic, if we don't have a parameter we shouldn't be including it in the params at all.